mantaray/received.py

```diff
@@ -312,13 +327,27 @@ def _handle_cap(server_view: views.ServerView, args: list[str]) -> None:
     if subcommand == "ACK":
         acknowledged = set(args[-1].split())
         if "sasl" in acknowledged:
+            server_view.core.cap_list.add("sasl")
             server_view.core.send("AUTHENTICATE PLAIN")
+        if "away-notify" in acknowledged:
+            server_view.core.cap_list.add("away-notify")
```

Comment:
Can we just add all acknowledged capabilities, instead of checking each one manually?